modules/nodes-pods-autoscaling-creating-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-creating-web-console_{context}"]
+
+= Creating a horizontal pod autoscaler by using web console
+
+From the web console, you can create a horizontal pod autoscaler that specifies the minimum and maximum number of pods you want to run on a deployment. You can also define the amount of CPU or memory use that your pods should target..
+
+[NOTE]
+====
+Horizontal pod autoscaler cannot be added to those Deployment or Deployment Configs that are part of operator-backed service, Knative service, or Helm chart.
+====
+
+.Procedure
+
+To create a horizontal pod autoscaler in the web console:
+
+. In the *Topology* view, click the node to reveal the side pane.
+. From the *Actions* drop-down list, select the *Add HorizontalPodAutoscaler* to open the *Add HorizontalPodAutoscaler* form.
++
+.Add HorizontalPodAutoscaler
+image::node-add-hpa-action.png[Add HorizontalPodAutoscaler form]
+. From the *Add HorizontalPodAutoscaler* form, define the name, minimum and maximum pod limits, the CPU and memory use, and click *Save*.
++
+[NOTE]
+====
+If any of the values for CPU and memory use are missing, a warning is displayed.
+====
+
+To edit a horizontal pod autoscaler in the web console:
+
+. In the *Topology* view, click the node to reveal the side pane.
+. From the *Actions* drop-down list, select the *Edit HorizontalPodAutoscaler* to open the *Edit Horizontal Pod Autoscaler* form.
+. From the *Edit Horizontal Pod Autoscaler* form, edit the minimum and maximum pod limits, the CPU and memory use, and click *Save*.
```

Comment:
s/limits, the CPU and memory use/limits and the CPU and memory usage

Reply:
Done